NuRadioReco/modules/RNO_G/hardwareResponseIncorporator.py

```diff
@@ -1,20 +1,23 @@
 from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.utilities import units, fft
+
 from NuRadioReco.detector.RNO_G import analog_components
+import NuRadioReco.detector.generic_detector as gdetector
```

Comment:
My preference would be to not invent new abbreviations if not necessary, so I would prefer to stick with generic_detector, but this is a minor nitpick as usual.

Reply:
This was taken from simulation.py. And I agree.